PresentationLayer/Sources/PresentationLayer/Features/NextToGo/NextToGoView.swift

```diff
@@ -123,6 +142,114 @@ private extension NextToGoView {
             }
         }
     }
+
+    @ViewBuilder
+    var headingStack: some View {
+        HStack(spacing: 16) {
+            Spacer()
+            topCountSelectionMenu
+            countrySelectionMenu
+        }
+    }
+
+    @ViewBuilder
+    var topCountSelectionMenu: some View {
+        Menu {
+            Picker("", selection: $viewModel.selectedTopCount) {
+                ForEach(viewModel.topCounts, id: \.self) {
+                    Text("\($0)")
+                }
+            }
+        } label: {
+            menuLabel(from: "🔝 \(viewModel.selectedTopCount)")
+                .accessibilityLabel(
+                    // TODO: 🤓 move logic into ViewModel, use localised copy and unit test it
+                    "Top \(viewModel.selectedTopCount) races"
```

Comment:
Left a note for myself as a reminder. Working well overall, checked on a phone, sounding nice.. :)